enyojs/ares-project
# ENYO-2820: Enable/disable minification from the "Phonegap build" UI

ENYO-2820 : The Ares developer can now enable/disable the minification from the panel "Phonegap Build", by activating/desactivating the checkbox "Activate the build minification" in the drawer "Build option".

How to test: 
1 - import a project based on a bootplate in Ares
2 - Open the "Phonegap Build" Panel and enable the checkbox "Activate the build minification"
3 - Run a build.
4 - check the size of the built application that are located in the directory "target/phonegap" of the project
5 - Open the "Phonegap Build" Panel and disable the checkbox "Activate the build minification"
6 - Run a build
7 - check the size of the built application that are located in the directory "target/phonegap" of the project

The size of each built application from step (4) should be inferior to those of the step (7) respectively.

Tested on Chrome (Windows 7)

Enyo-DCO-1.1-Signed-off-by: Rafik Adiche mahmoud-rafik.adiche@hp.com
